### PR TITLE
Add tutorial example

### PR DESCRIPTION
**Pull request by sacnl**

### CODE REVIEW

**Comment by sacnl** [comment]
Feel free to review, otherwise I'll merge it to master tomorrow and then mention the repo in the blog posts where appropriate.

File: SDK-Tutorial-CF/application/pom.xml [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+            <groupId>com.sap.cloud.s4hana.cloudplatform</groupId>
+            <artifactId>scp-cf</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>com.sap.cloud.s4hana</groupId>
+            <artifactId>s4hana-all</artifactId>
+        </dependency>
+        
+        <!-- Authentication and Authorization imports with Spring Security -->
+		<dependency>
```
Comment (sawoz) [inline SDK-Tutorial-CF/application/pom.xml]: just minor: indentation

File: SDK-Tutorial-CF/application/src/main/java/com/sap/cloud/s4hana/tutorial/ODataV2SimpleErrorHandler.java [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+import org.slf4j.Logger;
+
+import com.sap.cloud.sdk.cloudplatform.logging.CloudLoggerFactory;
+import com.sap.cloud.sdk.odatav2.connectivity.ErrorResultHandler;
+import com.sap.cloud.sdk.odatav2.connectivity.ODataException;
+
+public class ODataV2SimpleErrorHandler implements ErrorResultHandler<ODataException> {
+    private static final Logger logger = CloudLoggerFactory.getLogger(ODataV2SimpleErrorHandler.class);
+
+	public ODataException createError(String content, Object origin, int httpStatusCode) {
```
Comment (sawoz) [inline SDK-Tutorial-CF/application/src/main/java/com/sap/cloud/s4hana/tutorial/ODataV2SimpleErrorHandler.java]: Minor: code formatting

File: SDK-Tutorial-CF/application/src/main/webapp/WEB-INF/spring-security.xml [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+       xmlns:oauth="http://www.springframework.org/schema/security/oauth2"
+       xmlns:sec="http://www.springframework.org/schema/security"
+       xsi:schemaLocation="http://www.springframework.org/schema/security/oauth2
+        http://www.springframework.org/schema/security/spring-security-oauth2-1.0.xsd
+        http://www.springframework.org/schema/security
+        http://www.springframework.org/schema/security/spring-security-3.2.xsd
+        http://www.springframework.org/schema/beans
+        http://www.springframework.org/schema/beans/spring-beans-3.1.xsd">
+
+    <sec:http pattern="/**" create-session="never"
```
Comment (sawoz) [inline SDK-Tutorial-CF/application/src/main/webapp/WEB-INF/spring-security.xml]: Minor: code formatting

**Pull request merged**